internal/errcheck/errcheck_test.go

```diff
@@ -83,6 +90,94 @@ func TestAll(t *testing.T) {
 	test(t, CheckAsserts|CheckBlank)
 }
 
+const (
```

Comment:
Actually 2 constants here (`testVendorMain` and `testLog`), but don't feel strongly, so removed block.